### PR TITLE
Skip some WC FAT 3.1 and FAT 4 on EE10

### PR DESCRIPTION
**Pull request by pmd1nh**

### CODE REVIEW

**Comment by pmd1nh** [comment]
#build

**Review by pnicolucci — APPROVED** [review]

File: dev/com.ibm.ws.webcontainer.servlet.3.1_fat/fat/src/com/ibm/ws/webcontainer/servlet31/fat/tests/WCServerHttpUnit.java [hunk 1]
```diff
@@ -237,7 +240,8 @@ public void testRelativeRedirect() throws Exception {
 
     @Test
     @Mode(TestMode.FULL)
-    public void testRelativeRedirectWithPathInfo() throws Exception {
+    @SkipForRepeat(EE10_FEATURES)
+    public void testrelativeredirectwithpathinfo() throws Exception {
```
Comment (pnicolucci) [inline dev/com.ibm.ws.webcontainer.servlet.3.1_fat/fat/src/com/ibm/ws/webcontainer/servlet31/fat/tests/WCServerHttpUnit.java]: Why is this test method renamed?

**Review by pnicolucci — CHANGES_REQUESTED** [review]
Revert the change to the test method name. Thanks!

File: dev/com.ibm.ws.webcontainer.servlet.3.1_fat/fat/src/com/ibm/ws/webcontainer/servlet31/fat/tests/WCServerHttpUnit.java (hunk 1 above)
Comment (pmd1nh) [reply dev/com.ibm.ws.webcontainer.servlet.3.1_fat/fat/src/com/ibm/ws/webcontainer/servlet31/fat/tests/WCServerHttpUnit.java]: eclipse auto correction.  good catch

**Review by pnicolucci — APPROVED** [review]

**Comment by pnicolucci** [comment]
#libby

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#build

**Comment by pmd1nh** [comment]
#build

**Comment by pmd1nh** [comment]
#build

**Comment by pmd1nh** [comment]
Part2:
https://github.com/OpenLiberty/open-liberty/pull/20932

**Pull request merged**